File: 프로그래머스/unrated/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
  let [sum1, sum2] = [0, 0]

  if (arr1.length == arr2.length) {
    arr1.map(item => sum1 += item)
    arr2.map(item => sum2 += item)

    return sum1 > sum2 ? 1 : sum1 == sum2 ? 0 : -1
  }

  return arr1.length > arr2.length ? 1 : -1
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.java
class Solution {
  public int solution(int[] arr1, int[] arr2) {
    int sum1 = 0, sum2 = 0;
    
    if (arr1.length > arr2.length) return 1;
    else if (arr1.length < arr2.length) return -1;

    for (int i = 0; i < arr1.length; i++) {
      sum1 += arr1[i];
      sum2 += arr2[i];
    }
    
    if (sum1 > sum2) return 1;
    else if (sum1 < sum2) return -1; 
    else return 0;
  }
}